convert.py

```diff
@@ -114,29 +115,38 @@ def GenerateAudioContentForText(text):
         else:
             break
     
-
 def eupub_to_chapters():
+    """Parse a .epub file and return an array of strings
+    corresponding to the chapters of the epub.
+
+    Returns:
+        list(str): A list of string objects, each of which corresponds to 
+        a chapter of the .epub file, i.e. ["chapter 1 text", "chapter 2 text", ...]
+    """    
     path = os.path.abspath(EPUB_FILEPATH)
     print(path)

```

Comment:
Recommend either taking this out or making a more informative print statement, e.g.:

print("Reading EPUB from filepath: " + path)